action.yml

```diff
@@ -213,6 +213,10 @@ inputs:
     description:
       'The path to mount the workspace inside the docker container. For windows, leave out the drive letter. For example
       c:/github/workspace should be defined as /github/workspace'
+  quit:
```

Comment:
The input here is `quit` but in the rest of the code is `shouldQuite`. Is this ok?

Reply:
Thanks!